modules/mediagoBidAdapter.js

```diff
@@ -0,0 +1,364 @@
+/**
+ * gulp serve --modules=mediagoBidAdapter --nolint   --notest
+ */
+
+import * as utils from '../src/utils.js';
+import { getStorageManager } from '../src/storageManager.js';
+import {
+  registerBidder
+} from '../src/adapters/bidderFactory.js';
+
+const BIDDER_CODE = 'mediago';
+// const PROTOCOL = window.document.location.protocol;
+const ENDPOINT_URL =
+  // ((PROTOCOL === 'https:') ? 'https' : 'http') +
+  'http://ec2-44-233-123-136.us-west-2.compute.amazonaws.com:8081/api/bid?tn=';
```

Comment:
you cannot have a nonsecure endpoint

Reply:
Sorry, I think I could use test EP first to test if any other error. Now it has been changed to secure prod EP.